lib/auth/auth_with_roles.go

```diff
@@ -2383,14 +2383,19 @@ func (a *ServerWithRoles) GetSessionRecordingConfig(ctx context.Context, opts ..
 }
 
 // SetSessionRecordingConfig sets session recording configuration.
-func (a *ServerWithRoles) SetSessionRecordingConfig(ctx context.Context, recConfig types.SessionRecordingConfig) error {
-	if err := a.action(apidefaults.Namespace, types.KindSessionRecordingConfig, types.VerbCreate); err != nil {
+func (a *ServerWithRoles) SetSessionRecordingConfig(ctx context.Context, newRecConfig types.SessionRecordingConfig) error {
+	storedRecConfig, err := a.authServer.GetSessionRecordingConfig(ctx)
+	if err != nil {
```

Comment:
Just to ensure: the default SessionRecording values area always inserted thus `GetSessionRecordingConfig` call will never return NotFound error, right ?

Reply:
They are not inserted in the sense of being a hard-coded return value, but *some* `SessionRecordingConfig` value must be stored in the backend after auth server init.